tests/validator/cell-test.js

```diff
@@ -4,7 +4,7 @@ const expect = require('chai').expect
 const validatorFactory = require('../../lib/validator/cell')
 
 describe('validator/cell', () => {
-  let validator, cell, result, cellDefinitions, model, renderers
+  var validator, cell, result, cellDefinitions, model, renderers
```

Comment:
Our CI is testing Node 5 for some reason and I was getting complaints about `let`. This one may be fine but I got tired of waiting on Travis so I fixed a few at once